packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -23,13 +23,19 @@ import type {
   SchemaType,
   NativeModuleEnumMap,
   OptionsShape,
+  EventTypeAnnotation,
 } from '../CodegenSchema.js';
 
 import type {Parser} from './parser';
 import type {ParserType} from './errors';
 import type {ParserErrorCapturer, TypeDeclarationMap, PropAST} from './utils';
 import type {ComponentSchemaBuilderConfig} from './schema.js';
 
+type ObjectTypeAnnotationShape = {
+  type: 'ObjectTypeAnnotation',
+  properties: Array<NamedShape<EventTypeAnnotation>>,
+};
+
```

Comment:
We already have this in the [CodegenSchema](https://github.com/facebook/react-native/blob/main/packages/react-native-codegen/src/CodegenSchema.js#L53-L59). Could you reuse that instead of redefine the type, please? 🙏